packages/sdk/src/services/holograph-protocol.service.ts

```diff
@@ -204,4 +207,80 @@ export class HolographProtocol {
         throw new Error('This type of contract is not currently supported.')
     }
   }
+
+  async hydrateNFT(hydrateNftInput: {
+    chainId: number
+    address: Address
```

Comment:
I think calling this as `contractAddress` will make things more clear to the users